0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -275,21 +275,21 @@ For a detailed description of these properties, please see [`Message`](#class-me
 
 
 #### fulfillCondition
-<code>ledgerPlugin.fulfillCondition( **transferId**:String, **fulfillment**:String ) ⇒ Promise.&lt;null></code>
+<code>ledgerPlugin.fulfillCondition( **transferId**:String, **fulfillment**:String, **ilp**:String ) ⇒ Promise.&lt;null></code>
 
 Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promise should reject.
 
-The `fulfillment` is an arbitrary 32-byte buffer and is provided as a base64url-encoded string.
+The `fulfillment` is an arbitrary 32-byte buffer and is provided as a base64url-encoded string. `ilp` is an optional base64url-encoded ILP data packet that MAY be submitted alongside the fulfillment.
```

Comment:
Which [packet type](https://github.com/interledger/rfcs/blob/master/asn1/InterledgerPacket.asn#L30-L37) would this be?

Reply:
It would be a new one that would just be an `OCTET STRING` with data, but I don't think that the LPI needs to specify that